.github/workflows/load_tests.yml

```diff
@@ -0,0 +1,18 @@
+name: k6 Load Tests
+on:
+  schedule:
+  # Mon-Fri 5AM Finnish time (UTC+3)
+  - cron: '0 8 * * 1-5'
```

Comment:
5am finnish time is actually 2AM UTC-0 time?

Reply:
good point, fixed